riocli/apply/manifests/deployment-nonros-device.yaml

```
@@ -23,4 +23,7 @@ spec:
   volumes:
     - execName: exec-docker
       mountPath: "/tmp" # Path on container
-      subPath: "/tmp" # Path on device
\ No newline at end of file
+      subPath: "/tmp" # Path on device
+      uid: 1000 # userid for subpath
+      gid: 1000 # groupid for subpath
+      perm: 755 # permissions for subpath
```

suggestion: Consider marking these fields as optional so its clear that its not required.
```suggestion
      uid: 1000 # [Optional] userid for subpath
      gid: 1000 # [Optional] groupid for subpath
      perm: 755 # [Optional] permissions for subpath
```